python/pacemaker/_cts/watcher.py

```diff
@@ -53,7 +53,6 @@ def __init__(self, filename, host=None, name=None):
         host     -- The cluster node on which to watch the log
         name     -- A unique name to use when logging about this watch
         """
-        self.cache = []
```

Comment:
This commit gets mostly undone later. At the time, I wasn't sure what the final approach would be.

It doesn't hurt to leave this one in, as it gives a sort of clean slate for the later use of `self._cache`.